tests/data/preview/remove_for_brackets.py

```diff
@@ -29,6 +81,21 @@
         module._verify_python3_env = lambda: None
 
 # Brackets remain for long for loop lines
+for (
+    one_super_long_name_as_the_for_target_list_because_why_not_you_dont_control_me
+) in points:
+    pass
+
+for one_super_long_name_as_the_for_target_list_because_why_not_you_dont_control_me_i_have_total_freedom in (points):
```

Comment:
The odd parentheses is a pre-existing bug in main. Something in `right_hand_split` or something similar is deciding to make the invisible parentheses around `points` visible even though the split failed. I'm happy to try to fix this before merging, but I probably won't get to it until September.